client/app/scripts/superdesk-authoring/authoring.js

```diff
@@ -1431,6 +1431,8 @@
             },
             templateUrl: 'scripts/superdesk-authoring/views/send-item.html',
             link: function sendItemLink(scope, elem, attrs) {
+                var parentScope = scope.$parent.$parent;
```

Comment:
Hm... so how does the `monitoring` controller end up in the parent scope then, what places it there? Would it be possible to somehow reference that thing and obtain the controller reference from it?

@petrjasek @amagdas Are there any other alternatives to obtaining the controller from the grandparent scope?